binary_gentoo/internal/cli/build.py

```diff
@@ -174,8 +206,13 @@ def build(config):
         f'CFLAGS={shlex.quote(config.cflags)}',
         f'CXXFLAGS={shlex.quote(config.cxxflags)}',
         f'LDFLAGS={shlex.quote(config.ldflags)}',
+        f'USE={shlex.quote(config.use)}'
     ]
 
+    if config.tag_docker_image:
+        config.enforce_installation = True
+    container_name = f'gentoo-build-host-{uuid.uuid4().hex}'
```

Comment:
With my picky hat on: The "-host" part seems weird to me, because the container is not a host and the image is not either. Would you mind dropping that word? `f'binary-gentoo-{uuid.uuid4().hex}'` or `f'gentoo-{uuid.uuid4().hex}'` would also work well for me, just in case. Maybe shorter is nicer for logs. Just my 2 cents.

Reply:
Good point. Modified to `binary-gentoo` to make it obvious it is linked to the `binary-gentoo` project.

Reply:
Okay cool